example/states/webview/lerna.json

```diff
@@ -1,11 +1,13 @@
 {
-  "lerna": "2.4.0",
-  "version": "0.0.0",
+  "version": "independent",
   "useWorkspaces": true,
   "npmClient": "yarn",
   "command": {
     "run": {
       "stream": true
     }
-  }
-}
\ No newline at end of file
+  },
+  "packages": [
+    "states-sprotty*"
+  ]
+}
```

Comment:
Could we get rid of Lerna here and move the content of the `states-sprotty-webview` subfolder one level higher?

Reply:
Merged the two webview packages into one and removed lerna